test/compute-engine/simplify.test.ts

```diff
@@ -128,45 +128,6 @@ const CANONICALIZATION_TEST_CASES: TestCase[] = [
               //
             `,
   ],
-  ['x/(y/2)^3', '(8*x)/y^3'],
```

Comment:
Why did those test cases were moved to RULE_TEST_CASE?

Reply:
It didn't need to be removed. I didn't realize t the fact that the RULE_TEST_CASE only includes stuff related to the rules because there are many cases there that don't use any of the rules

Reply:
OK, I'll move them back. The cases that don't use any of the rules should also be moved out of RULE_TEST_CASE but that can be done later.